pubsub/types/pubsub_types.go

```diff
@@ -38,15 +39,27 @@ func IsTemporaryErr(err error) bool {
 // WebSocketMessage represents the JSON object used to send and receive typed
 // messages to the web client.
 type WebSocketMessage struct {
-	EventId string `json:"event"`
-	Message string `json:"message"`
+	EventId string          `json:"event"`
+	Message json.RawMessage `json:"message"`
```

Comment:
Raw JSON because the Message will be Marshalled/Unmarshalled from different types, below.